packages/runtime/src/services/StateManager.ts

```diff
@@ -169,18 +171,10 @@ export class StateManager {
     options: EvalOptions = {}
   ): EvaledResult<T> {
     const store = this.slotStore;
-    const redirector = new Proxy(
-      {},
-      {
-        get(_, prop) {
-          return options.slotKey ? store[options.slotKey][prop] : undefined;
-        },
-      }
-    );
 
     options.scopeObject = {
       ...options.scopeObject,
-      $slot: redirector,
+      $slot: options.slotKey ? store[options.slotKey] : undefined,
```

Comment:
The original way pass a `Proxy` to `$slot`. But `$slot` doesn't work in module if using Proxy. So I pass the value in store as `$slot` to fix this bug.